apps/site/assets/ts/helpers/date.ts

```diff
@@ -77,4 +83,22 @@ export const formatToBostonTime = (
   return formatInTimeZone(dateTime, "America/New_York", formatString);
 };
 
+export const formatRelativeTime = (
+  time: Date,
+  targetDate: Date = new Date()
+): string => {
+  const seconds = differenceInSeconds(time, targetDate);
+  if (seconds <= 30) {
+    return "Arriving";
+  }
+  if (seconds <= 60) {
+    return "<1 minute away";
```

Comment:
Adding clarity to previous comment, based on the ticket I think this needs a paramter to distinguish if this is a schedule or a prediction. If it's a schedule the text should be "<1 minute away" and if it's a prediction it should be "approaching"

Reply:
Thanks for raising -- after talking it over with Anna we've decided to keep both prediction and schedule as "<1 minute away" for now.